indico/modules/groups/controllers.py

```diff
@@ -52,7 +60,8 @@ def _process(self):
         provider_titles = {p.name: p.title for p in multipass.identity_providers.itervalues()}
         provider_titles[None] = _('Local')
         return WPGroupsAdmin.render_template('groups.html', groups=groups, providers=providers, form=form,
-                                             search_results=search_results, provider_titles=provider_titles)
+                                             search_results=search_results, provider_titles=provider_titles,
+                                             groups_enabled=groups_enabled, local_groups=local_groups)
```

Comment:
`local_groups` is not needed: in templates we have a global `indico_config` so you can access `indico_config.LOCAL_GROUPS` there